pkg/controller/command/verifiable/models.go

```diff
@@ -55,8 +53,8 @@ type ProofOptions struct {
 	SignatureType string `json:"signatureType,omitempty"`
 	// PrivateKey is used to sign instead of DID
 	PrivateKey string `json:"privateKey,omitempty"`
```

Comment:
there should be a followup to remove this.

Reply:
created : https://github.com/hyperledger/aries-framework-go/issues/1748